src/main/java/org/jlab/myquery/IntervalWebService.java

```diff
@@ -109,22 +119,18 @@ private EventStream<FloatEvent> doApplicationSampling(String sampleType, Metadat
     }
 
     private EventStream<FloatEvent> doDatabaseSourceSampling(String sampleType, Metadata<FloatEvent> metadata, boolean updatesOnly, Instant begin, Instant end, long limit,
-                                                 FloatEvent priorEvent, Class<FloatEvent> type) throws SQLException {
+                                                             FloatEvent priorEvent, Class<FloatEvent> type) throws SQLException {
         EventStream<FloatEvent> stream;
 
-        switch(sampleType) {
+        switch (sampleType) {
```

Comment:
Single case statement in a switch is candidate for if/else instead.

Reply:
Fixed.